### PR TITLE
Add CI to lava-dl

### PR DESCRIPTION
**Pull request by mgkwill**

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+use_plugin('python.install_dependencies')
+use_plugin("python.pycharm")
+use_plugin('pypi:pybuilder_bandit')
+
+name = "lava-nc/lava-dl"
+default_task = ["analyze", "publish"]
+version = "0.2.0"
+summary = "A library of deep learning tools, which consists of lava.lib.dl.slayer and lava.lib.dl.netx for training and deployment of event-based deep neural networks on traditional as well as neuromorphic backends. Lava-DL is part of Lava Framework "
+url = "https://lava-nc.org"
+license = ["BSD-3-Clause", "LGPL-2.1"]
```
Comment (bamsumit) [inline build.py:19]: BSD-3 only license

Comment (mgkwill) [reply build.py]: Thanks. Done!

**Review by mgkwill — COMMENTED** [review]
Looks good to me.

**Review by bamsumit — APPROVED** [review]
Looks good. Thanks

**Comment by mgkwill** [comment]
Unit tests will be added on @bamsumit fork for 0.2.0 release

**Pull request merged**